gitpg/williams/langtonant.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
```

Comment:
Hi Williams, solution looks clean and interesting, however, we need to run this using node js, could you please modify your solution so it doesn't need to use an html file? it is not required to show each step, just print final grid status after the steps taken by the ant and this can be done by printing in the console